guides/plugins/plugins/administration/add-new-tab.md

```diff
@@ -223,13 +223,13 @@ As shown in the previous example, your custom component is expected to be in a d
 
 ```text
 administration
-├── page
-│   └── sw-product-detail
+├── src
+│   └──page
```

Comment:
@seggewiss , The actual path seems to be this `src/Administration/Resources/app/administration/src/module/sw-product/page/sw-product-detail/sw-product-detail.html.twig` . Should I include also include `modules/sw-product` after src directory?